### PR TITLE
What are those to be added ?

### PR DESCRIPTION
**Pull request by sokratis12GR**

### CODE REVIEW

File: src/main/java/com/mcmoddev/communitymod/sokratis12gr/whatarethose/WhatAreThose.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.regex.Pattern;
+
+@SubMod(
+    name = "What are Those",
+    description = "WHAT ARE THOOOOOSE!?",
+    attribution = "sokratis12GR"
+)
+@Mod.EventBusSubscriber
+public class WhatAreThose implements ISubMod {
+    private static Pattern neatPattern = Pattern.compile("hi", Pattern.CASE_INSENSITIVE);
```
Comment (antoinegag) [inline src/main/java/com/mcmoddev/communitymod/sokratis12gr/whatarethose/WhatAreThose.java]: why are you matching "hi" 🤔

Comment (sokratis12GR) [reply src/main/java/com/mcmoddev/communitymod/sokratis12gr/whatarethose/WhatAreThose.java]: well, tbh didn't know what to match... any suggestions ?

Comment (antoinegag) [reply src/main/java/com/mcmoddev/communitymod/sokratis12gr/whatarethose/WhatAreThose.java]: Fire it when you equip boots that are under diamond 😂

Comment (sokratis12GR) [reply src/main/java/com/mcmoddev/communitymod/sokratis12gr/whatarethose/WhatAreThose.java]: actually, was thinking something similar, but nice idea!

File: src/main/java/com/mcmoddev/communitymod/sokratis12gr/whatarethose/WhatAreThose.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    @SubscribeEvent()
+    public static void onEquipEvent(TickEvent.PlayerTickEvent event) {
+        if (rand.nextInt(1000) < 1) {
+            EntityPlayer player = event.player;
+            BlockPos pos = new BlockPos(player.posX, player.posY, player.posZ);
+            IBlockState state = player.world.getBlockState(pos.add(0, -1, 0));
+            if (player.onGround) {
+                for (ItemStack stack : player.inventory.armorInventory) {
+                    if (stack.getItem() instanceof ItemArmor && stack.getEquipmentSlot() == EntityEquipmentSlot.FEET && state == Blocks.DIAMOND_BLOCK) {
```
Comment (antoinegag) [inline src/main/java/com/mcmoddev/communitymod/sokratis12gr/whatarethose/WhatAreThose.java:35]: When I said "under diamond" I meant "The tier is under diamond" (so if you equip Iron or Leather boots). Not literally under a block of diamond lol

Comment (sokratis12GR) [reply src/main/java/com/mcmoddev/communitymod/sokratis12gr/whatarethose/WhatAreThose.java:35]: Welp, I got you the wrong way...

**Pull request merged**